src/app/views/events-page/events-page.component.html

```diff
@@ -74,6 +94,39 @@
         </div>
     </div>
 
+    <div *ngIf="eventModal">
+        <div class="modal">
+        <div class="ui active modal">
+            <i class="close icon" (click)="hideEventModal()"></i>
+            <div class="header">EVENT: {{clickedEvent.title}}</div>
+            <div class="tableContent">
+                <p>DATE: {{clickedDate}}</p>
+                <p>TIME: {{clickedEvent.startTime}} to {{clickedEvent.endTime}}</p>
+            </div>
+            <div class="actions">
+            </div>
+        </div>
+        </div>
+    </div>
+
+    <div class="content-block">
+    <div class="ui card centered">
+        <div class="content">
+            <full-calendar 
+            defaultView="dayGridMonth"
+            [plugins]="calendarPlugins"
+            [events]="calendarEvents"
+            (eventClick)="handleEventClick($event)"
+            (dateClick)="handleClick($event)"
+            ></full-calendar>
+        </div>
+        <div class="content">
+            <button (click)="displayModal()" class="ui button">Schedule event</button>
```

Comment:
Do you think we should move this to the top of the calendar? That way it's easier for the user to see.